library/lua/argparse.lua

```diff
@@ -141,7 +162,13 @@ function stringList(arg, arg_name, list_length)
     return list
 end
 
+---@nodiscard
+---@param arg string
+---@param arg_name? string
+---@param list_length? integer
+---@return integer[]
 function numberList(arg, arg_name, list_length)
+    ---@type integer[]
```

Comment:
ugh, should we just declare a new output table so we don't have to live this lie?

Reply:
Sure can do!